app/scripts/metamask-controller.js

```diff
@@ -2,9 +2,10 @@ import EventEmitter from 'events';
 import pump from 'pump';
 import { ObservableStore } from '@metamask/obs-store';
 import { storeAsStream } from '@metamask/obs-store/dist/asStream';
-import { JsonRpcEngine } from 'json-rpc-engine';
+import { JsonRpcEngine, createAsyncMiddleware } from 'json-rpc-engine';
```

Comment:
Dont need this dep anymore
```suggestion
import { JsonRpcEngine } from 'json-rpc-engine';
```